### DISCUSSION TITLE
Github student not approved.

### DISCUSSION THREAD

**Discussion by Timaki-Sasanoha** [community/community#77532] [GitHub Education]
### Hi everyone,

Hi, everyone.
I submitted  my informations to use GitHub student. In this page ' https://education.github.com/discount_requests/application ', displayed my request is approved. however, after approved, an email was sent to me "Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on ~~~". this mail is written that "Once you have resolved the issues with your sumbission, visit us again and reapply(https://education.github.com/)." but I cannot reapply because my request is approved. 
What should I do next?

Best regards

**Comment by hackelite01** [comment]
Hey @Timaki-Sasanoha , I can do it easily. Drop the details on mail at [hackelite.sup@gmail.com](mailto:hackelite.sup@gmail.com)
